api/v1alpha1/localbuild_types.go

```diff
@@ -20,7 +20,14 @@ type EmbeddedArgoApplicationsPackageConfigSpec struct {
 	Enabled bool `json:"enabled,omitempty"`
 }
 
+// GitConfigSpec controls what git server to use for the idpbuilder
+// It can take on the values of either gitea or gitserver
+type GitConfigSpec struct {
+	Type string `json:"type,omitempty"`
```

Comment:
Wish there was a way to put the reference to secrets that contain gitea admin credentials in Spec or Status. I think for this implementation we can just document secret requirements. Not a blocker imo, but we need to open an issue.

Reply:
well the secret name is hardcoded here, so maybe we can just use it for now. i like the idea https://github.com/cnoe-io/idpbuilder/pull/39/files#diff-6eb0197d5d12e1ca681eac36170032f5d01e997ea21bdbd87803a302f9df254fR582-R590